ui/raidboss/data/04-sb/raid/o7n.ts

```diff
@@ -23,12 +54,12 @@ const triggerSet: TriggerSet<Data> = {
       alertText: (_data, _matches, output) => output.text!(),
       outputStrings: {
         text: {
-          en: 'Magitek Ray',
-          de: 'Magitek-Laser',
-          fr: 'Rayon Magitek',
-          ja: '魔導レーザー',
-          cn: '直线AOE',
-          ko: '마도 레이저',
+          en: 'Get out of front',
+          de: 'Weg von vorn',
+          fr: 'Partez du devant',
+          ja: '正面から離れる',
+          cn: '远离正面',
+          ko: '정면 피하기',
```

Comment:
Huhhhh, we *don't* have a canned response for this. I'm honestly a little surprised!

Reply:
Oh hmm, we do have `awayFromFront`.  I'll use that.